react/components/ExtensionPoint/index.tsx

```diff
@@ -161,7 +161,7 @@ function withOuterExtensions(
     <Fragment key={`wrapped-${treePath}`}>
       <LazyImages>{beforeElements}</LazyImages>
       {element}
-      {isRootTreePath && <div className="flex flex-grow-1" />}
+      {isRootTreePath && <div className="flex flex-grow-1 bg-checkout-color" />}
```

Comment:
An ExtensionPoint is used in many situations, so the name should be more generic than this.

```suggestion
      {isRootTreePath && <div className="flex flex-grow-1 below-element" />}
```